src/ClusterBootstrap/params.py

```diff
@@ -62,7 +62,9 @@
         "prometheus-ip": "localhost",
         "prometheus-port": 9091,
         "etcd": {
-            "data-dir": "/etc/RepairManager/etcd"
+            "data-dir": "/etc/RepairManager/etcd",
+            "peer-port": 2381,
```

Comment:
Where is `peer-port` used?

Reply:
`peer-port` is used a couple times in the etcd.conf.yaml, which is used for starting up the etcd server.

https://github.com/microsoft/DLWorkspace/blob/e480f7f2f1868cb2ef5248363024d9ac14886e70/src/ClusterBootstrap/template/RepairManager/etcd.conf.yaml#L7

It's a required configuration to start up the etcd server, but it's not used otherwise (as there are currently no other etcd peers).